conda/build_pytorch.sh

```diff
@@ -242,6 +242,9 @@ else
     . ./switch_cuda_version.sh "$desired_cuda"
     # TODO, simplify after anaconda fixes their cudatoolkit versioning inconsistency.
     # see: https://github.com/conda-forge/conda-forge.github.io/issues/687#issuecomment-460086164
+    if [[ "$desired_cuda" == "11.0" ]]; then
```

Comment:
https://anaconda.org/anaconda/cudatoolkit/files Looks like it has not be released in anaconda.